core/src/mindustry/ui/fragments/HudFragment.java

```diff
@@ -293,6 +293,11 @@ public void build(Group parent){
         //core info
         parent.fill(t -> {
             t.top();
+
+            if(Core.settings.getBool("macnotch") ){
+                t.margin(32);
```

Comment:
Is `32` always the correct value for this? Does it vary by device and monitor settings?

Reply:
AFAIK, all notches are of the same height _currently_. 
And I didn't find a way to get notch dimensions dynamically in Java app on mac. 
I know that magic number are better to store as constants and reuse (I use this one twice), but not sure how to do it in Java, and in Mindustry project style.

Reply:
Maybe add a `public static float macNotchHeight = 32f` in `Vars`